prov/efa/src/rxr/rxr.h

```diff
@@ -64,9 +64,9 @@
 #include <ofi_perf.h>
 #include <ofi_hmem.h>
 
+#include "rxr_op_entry.h"
 #include "rxr_pkt_entry.h"
 #include "rxr_pkt_type.h"
-#include "rxr_op_entry.h"
```

Comment:
Hmmm not sure if this change's needed.

Reply:
It is necessary because `rxr_pkt_entry.h` and `rxr_pkt_type.h` referred to`rxr_tx_entry` and `rxr_rx_entry`.

Reply:
I see, thanks.